### PR TITLE
Client things and some others

### PR DESCRIPTION
**Pull request by amusingimpala75**
Mappings of the through out the client package, and them some that I managed to run through when mapping said classes. Let me know if you want smaller pr next time.

### CODE REVIEW

File: mappings/net/minecraft/RainSpalshParticle.mapping [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_260 net/minecraft/RainSpalshParticle
```
Comment (sschr15) [inline mappings/net/minecraft/RainSpalshParticle.mapping]: Typo? Also it's in the `net.minecraft` package?
```suggestion
CLASS net/minecraft/class_260 net/minecraft/client/entity/particle/RainSplashParticle
```

File: mappings/net/minecraft/client/render/ParticleManager.mapping [hunk 2]
```diff
@@ -1,9 +1,26 @@
...
+		ARG 2 y
+		ARG 3 z
+	METHOD method_645 addBlockBreakingParticles (IIII)V
+		COMMENT direction:
+		COMMENT \tDOWN: \t0
+		COMMENT \tUP:\t\t1
+		COMMENT \tNORTH:\t2
+		COMMENT \tSOUTH:\t3
+		COMMENT \tWEST: \t4
+		COMMENT \tEAST:\t5
```
Comment (sschr15) [inline mappings/net/minecraft/client/render/ParticleManager.mapping]: Inconsistent spaces / tabs after the comments...?
Would it work to instead use a table? Javadoc does support (some) HTML tags

**Review by sschr15 — APPROVED** [review]
Looks pretty good to me

File: mappings/net/minecraft/RainSpalshParticle.mapping (hunk 1 above)
Comment (amusingimpala75) [reply mappings/net/minecraft/RainSpalshParticle.mapping]: Yep, that's a typo

File: mappings/net/minecraft/client/render/ParticleManager.mapping (hunk 2 above)
Comment (amusingimpala75) [reply mappings/net/minecraft/client/render/ParticleManager.mapping]: I should just use a table. When I looked at it in Enigma, it was not lined up in the normal editor if it was lined up in the javadoc editor

File: mappings/net/minecraft/client/entity/particle/RainSpalshParticle.mapping [hunk 3]
```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_260 net/minecraft/client/entity/particle/RainSpalshParticle
```
Comment (sschr15) [inline mappings/net/minecraft/client/entity/particle/RainSpalshParticle.mapping]: typo?
```suggestion
CLASS net/minecraft/class_260 net/minecraft/client/entity/particle/RainSplashParticle
```

**Review by sschr15 — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/entity/particle/RainSpalshParticle.mapping (hunk 3 above)
Comment (amusingimpala75) [reply mappings/net/minecraft/client/entity/particle/RainSpalshParticle.mapping]: Yeah, that’s a typo. I need to get back to this after being distracted for a few months by game engines and c++.

**Review by sschr15 — APPROVED** [review]
now all i need to do is figure out how to fix the merge conflicts

**Comment by sschr15** [comment]
Merged in https://github.com/TextileMC/yarrn/commit/6b358f6d885c068abcddf7275764bf151d0943cd

**Pull request closed without merging**